cmd/release/main.go

```diff
@@ -0,0 +1,69 @@
+package main
```

Comment:
this is a nicer idea than my source grepping proposal
just make sure this is well documented so it's clear what this is doing in here and that it has a very narrow usecase

Reply:
On it, I'll add a README to that package and document the interface and the purpose of the CLI.